components/Action/src/index.scss

```diff
@@ -0,0 +1,115 @@
+.denhaag-action {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  gap: var(--denhaag-action-gap);
+  color: var(--denhaag-action-color);
+  background-color: var(--denhaag-action-background-color);
+  padding-inline-start: var(--denhaag-action-padding-inline-start);
+  padding-inline-end: var(--denhaag-action-padding-inline-end);
+  padding-block-start: var(--denhaag-action-padding-block-start);
+  padding-block-end: var(--denhaag-action-padding-block-end);
+  border-style: var(--denhaag-action-border-style);
+  border-width: var(--denhaag-action-border-width);
+  border-color: var(--denhaag-action-border-color);
+
+  @media (min-width: 640px) {
```

Comment:
Currently, we don't have any breakpoint on 640px, so we should probably use 768px. See https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/Den-Haag-Design-System?type=design&node-id=3909-0&mode=dev